dds_cli/__main__.py

```diff
@@ -211,6 +212,7 @@ def list_projects_and_contents(
                 show_usage=usage,
                 no_prompt=click_ctx.get("NO_PROMPT", False),
                 json=json,
+                show_all=show_all,
```

Comment:
Move this so that it says `projects = lister.list_projects(sort_by=sort, show_all=show_all)` instead
```suggestion
```